### ISSUE TITLE
Cant compile my flutter app with Xcode

### ISSUE THREAD

**Issue by NTMS2017** [zhangao0086/DKImagePickerController#662]
I can run my flutter app on iPhone using android studio with no problem. But when I run using Xcode it doesn't build. I tested every possible combination and I am keep getting below error.

I need to use Xcode to upload my app but Xcode error not allowing me to build at the moment.

I am not sure which flutter plugin has use _FLAnimatedImageView or similar.

Error:

Ld /Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Intermediates.noindex/Pods.build/Release-iphoneos/DKPhotoGallery.build/Objects-normal/arm64/DKPhotoGallery normal arm64
    cd /Users/***************/IdeaProjects/my_app/ios/Pods
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -Xlinker -rpath -Xlinker /usr/lib/swift -target arm64-apple-ios9.0 -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.5.sdk -L/Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Products/Release-iphoneos/DKPhotoGallery -F/Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Products/Release-iphoneos/DKPhotoGallery -F/Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Products/Release-iphoneos/FLAnimatedImage -F/Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Products/Release-iphoneos/SDWebImage -F/Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Products/Release-iphoneos/SDWebImageFLPlugin -filelist /Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Intermediates.noindex/Pods.build/Release-iphoneos/DKPhotoGallery.build/Objects-normal/arm64/DKPhotoGallery.LinkFileList -install_name @rpath/DKPhotoGallery.framework/DKPhotoGallery -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Intermediates.noindex/Pods.build/Release-iphoneos/DKPhotoGallery.build/Objects-normal/arm64/DKPhotoGallery_lto.o -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Intermediates.noindex/Pods.build/Release-iphoneos/DKPhotoGallery.build/Objects-normal/arm64/DKPhotoGallery.swiftmodule -framework AVFoundation -framework AVKit -framework Foundation -framework Photos -framework UIKit -framework AVFoundation -framework AVKit -framework Foundation -framework Photos -framework SDWebImage -framework SDWebImageFLPlugin -framework UIKit -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Intermediates.noindex/Pods.build/Release-iphoneos/DKPhotoGallery.build/Objects-normal/arm64/DKPhotoGallery_dependency_info.dat -o /Users/***************/Library/Developer/Xcode/DerivedData/Runner-daegbpqmxfekohfoqvekeirskbko/Build/Intermediates.noindex/Pods.build/Release-iphoneos/DKPhotoGallery.build/Objects-normal/arm64/DKPhotoGallery

Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_FLAnimatedImageView", referenced from:
      type metadata for DKPhotoGallery.DKPhotoImageView in DKPhotoImageView.o
  "_OBJC_METACLASS_$_FLAnimatedImageView", referenced from:
      _OBJC_METACLASS_$__TtC14DKPhotoGallery16DKPhotoImageView in DKPhotoImageView.o
  "_OBJC_CLASS_$_FLAnimatedImage", referenced from:
      objc-class-ref in DKPhotoBaseImagePreviewVC.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by zhangao0086** [comment]
Have you tried to **Clean** and **Clean Build Folder** and rebuild?

**Comment by NTMS2017** [comment]
Yes I did. Cmd + shift + K didn't work. I am not sure which flutter plugin has use _FLAnimatedImageView or similar. In my flutter app I didn't use any plugin that has FLAnimatedImageView. 

I am not sure what else I can do?

**Comment by zhangao0086** [comment]
Replaced FLAnimatedImageView with SwiftyGif 😄  Please `pod update` 😃
